akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -219,13 +219,14 @@ class IntroSpec extends ScalaTestWithActorTestKit with WordSpecLike {
           chatRoom ! GetSession("ol’ Gabbler", gabblerRef)
 
           Behaviors.receiveSignal {
-            case (_, Terminated(ref)) =>
+            case (_, Terminated(_)) =>
               Behaviors.stopped
           }
         }
 
       val system = ActorSystem(main, "ChatRoomDemo")
       //#chatroom-main
+      system.terminate() // remove compiler warnings
```

Comment:
if someone tries to run this (copying from source) it might terminate too early, and expected logging not shown